lib/payment_processor/clients/braintree/transaction.rb

```diff
@@ -3,18 +3,33 @@ module Clients
     module Braintree
       class Transaction
 
-        def self.make_transaction(nonce:, amount:, user:)
-          new(nonce, amount, user).sale
+        def self.make_transaction(nonce:, amount:, user:, store: nil)
+          new(nonce, amount, user, store).sale
```

Comment:
Didn't realize to ask this earlier, but what's store? It just gets passed as nil here so I'm not sure I understand what it represents exactly.